src/deploy.sh

```diff
@@ -96,7 +96,8 @@ function deploy_main()
     exit 22 # EINVAL
   fi
 
-  if [[ -z "${options_values['FROM_PACKAGE']}" ]]; then
+  if [[ -z "${options_values['FROM_PACKAGE']}" &&
```

Comment:
We can use `CAN_RUN_OUTSIDE_KERNEL_TREE` to replace `FROM_PACKAGE` as well.

Reply:
Tested and done 👍